src/renderer/template.ts

```diff
@@ -28,10 +34,22 @@ export async function renderTemplate(filepath: string, context: TemplateContext)
  * @private
  * @param filepath to import
  */
-function importComponent(filepath: string, context: TemplateContext): Promise<React.ReactElement> {
-  return new Promise((resolve) => {
-    import(filepath).then(component => { resolve(component.default !== undefined ? component.default(context) : undefined) })
-  })
+function importComponent(filepath: string, context: TemplateContext): Promise<React.ReactElement | undefined> {
+  return new Promise((resolve, reject) => {
+    try {
+      // we should import component only in NodeJS
+      if (require === undefined) resolve(undefined);
+      // remove from cache imported file
+      delete require.cache[require.resolve(filepath)];
+
+      const component = require(filepath);
+      if (typeof component === "function") resolve(component(context));
```

Comment:
Will this ever happen? 🤔 Or is this triggered by something like the following?
```
export function test(){}
```

Reply:
then it return `undefined`, and then we check if return value is `File` component, if yes, render template

Reply:
with require if you write `export default function test() {}` then `component` will be this function, in other way imported `component` value will be `object` type.

Reply:
Ahh I see 👍